lib/chef-dk/command/generator_commands/cookbook.rb

```diff
@@ -77,6 +85,7 @@ def initialize(params)
         def run
           read_and_validate_params
           if params_valid?
```

Comment:
I think this would be more clear as `if params_valid? && valid_cookbook_name?` and then strike the next line and return `1`